### PR TITLE
voice 3

### PR DESCRIPTION
**Pull request by maxmcd**

### CODE REVIEW

**Comment by kuanb** [comment]
Would love to know what codecov/patch and /project are...

File: app/lib/models.js [hunk 1]
```diff
@@ -14,18 +14,32 @@ function undefinedValuesCheck (array) {
...
+      ).update(this._cleanParams(params))
+      .returning("*")
+      .then((objs) => {
+        // TODO? update params on this instance
+        // and not just the new one so that
+        // if it is re-used they are correct?
+        this.constructor._getSingleResponse(objs, fulfill)
+      })      
+    })
+  }
```
Comment (kuanb) [inline app/lib/models.js]: so this requires that the primary Id be supplied in the params.
perhaps this is just me but would it be more explicit to have that primaryId be a second param? that way it is explicit

File: app/lib/voice.js [hunk 2]
```diff
@@ -24,30 +23,46 @@ module.exports = {
...
+          statusCallbackEvent: ["initiated", "ringing", "answered", "completed"],
+          StatusCallback: `${domain}/webhook/voice/status`,
+        }, function(err, call) {
+          if (err) {
+            reject(err)
+          } else {
+            ovm.update({call_sid: call.sid})
+            .then((ovm) => {
+              fulfill(ovm)
+            })
```
Comment (kuanb) [inline app/lib/voice.js]: no catch() after .then?

**Comment by kuanb** [comment]
both comments are opinions... otherwise lgtm

File: app/lib/models.js (hunk 1 above)
Comment (maxmcd) [inline app/lib/models.js]: This is an instance method, not a class method. So you would call it on `message.update`, so the id would already be present on the instance itself.

Comment (kuanb) [inline app/lib/models.js]: gotcha ok

**Comment by kuanb** [comment]
lgtm then

**Pull request merged**